### PR TITLE
Paginate games table in UI

### PR DESCRIPTION
**Pull request by MariusTulbure**
Fixes https://github.com/robert-damoc/game-stats/issues/22

image

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ def index
     @sort_dir = params[:sort_dir] || Game.default_direction
 
     @games = Game.all.sort_table(@sort, @sort_dir)
+    @pagy, @games = pagy(@games, items: 20)
```
Comment (robert-damoc) [inline app/controllers/games_controller.rb:10]: `20` here is called a magic number. You don't know where it comes from, and have to "investigate" to understand what it means. I suggest adding maybe to application controller something like
```
PER_PAGE = 20
```
and then use it here

**Review by robert-damoc — COMMENTED** [review]
Remove this file `config/.DS_Store`

**Comment by robert-damoc** [comment]
I was looking at the [pagy documentation](https://github.com/ddnexus/pagy), and saw we can make some changes to the defualts there ([link](https://github.com/ddnexus/pagy))

[pagy_info(@pagy)](https://ddnexus.github.io/pagy/docs/api/frontend) looks like something cool we can add above the table

I think it'd also be (maybe) helpful to add the per_page selector there: [pagy_items_selector_js](https://ddnexus.github.io/pagy/docs/api/javascript/)

**Comment by MariusTulbure** [comment]
About the [pagy_items_selector_js](https://ddnexus.github.io/pagy/docs/api/javascript/) I thinck it would not look good beacouse it comes in only one style which I, personally, do not realy like.
I inserted [pagy_info(@pagy)](https://ddnexus.github.io/pagy/docs/api/frontend).

image

**Comment by robert-damoc** [comment]
Ok, we can skip the [pagy_items_selector_js](https://ddnexus.github.io/pagy/docs/api/javascript/).

But for the info, I think that one should be above the games table

**Comment by MariusTulbure** [comment]
image
What do you think?
Should I style it a bit, how do you think I should do it?

**Review by robert-damoc — APPROVED** [review]
Looks good for now :+1:

**Pull request merged**